app/templates/orders/view.hbs

```diff
@@ -34,16 +34,18 @@
         </div>
       </div>
       <br>
-      <a class="ui labeled icon {{if isLoadingTickets 'loading'}} button" href="#" {{action downloadTickets model.event.name model.order.identifier}} target="_blank" rel="noopener nofollow">
-        <i class="ticket alternate icon"></i>
-        {{t 'Download Tickets'}}
-      </a>
-      <br>
-      <br>
-      <a href="#" {{action 'downloadInvoice' model.event.name model.order.identifier }} class="ui labeled icon blue {{if isLoadingInvoice 'loading'}} button">
-        <i class="print alternate icon"></i>
-        {{t 'Print Invoice'}}
-      </a>
+      {{#if (eq model.order.status 'completed')}}
```

Comment:
@uds5501 I think we are redirecting pending tickets to `/pending` rather than `/view`. Can you check?

Reply:
Ah! @uds5501 I don't see any `Download Tickets` option on the pending page.
![Uploading Screenshot_2019-05-26 Pending Order -5283ba75-7116-4cc6-b048-b54df9ec566a Orders Open Event.png…]()

Reply:
@shreyanshdwivedi We want download and invoices ONLY in completed orders, correct?

Reply:
Yes @uds5501. We want tickets only for `Completed` or `Placed` orders but for `Pending` tickets we are already not showing any option to download tickets (for reference - screenshot attached above).
`/view` route is accessed only by `Placed` and `Completed` orders

Reply:
Wait a second, we determined that placed orders aren't supposed to be generating tickets or invoices either, correct? @shreyanshdwivedi